_posts/2021-06-010-How-to-stop-getting-notifications.md

```diff
@@ -0,0 +1,158 @@
+  - {id: step-12, text: Notification preferences}
+  
+
+categories:
+  - How-To
+# set: getting-started
+# set_order: 3
+---
+
+Sometimes you're getting continous email notifications or you want to edit your preferences.
+
+To change notifications setting or disable any notification, follow the steps mentioned below
+
+
+
+
+## Step-1 
+
+How to stop getting all notifications.
+
+Go to "assistant", if you are Org admin you'll see all the assistants.
+
+![image](../../images/Stop-getting-notification-1.png)
+
+
+## Step-2
+
+Enable 'show my assistant only'.
+
+![image](../../images/Stop-getting-notification-2.png)
+
+## Step-3
+
+Click notification.
+
+![image](../../images/Stop-getting-notification-3.png)
+
+## Step-4
+
+You will be directed to the notification page and disable 'receive email notification'
+
+![image](../../images/Stop-getting-notification-4.png)
+
+Note: There is no way to change the notification for all the assistants in one click. So you have to go to each of the assistants and then disable the notification.
+
+## Step-5 
+
+If you want to stop specific notification from a particular assistant
+
+Click on the notification.
+
+![image](../../images/Stop-getting-notification-5.png)
+
+Here you'll see a list of all notifications.
+
+![image](../../images/Stop-getting-notification-6.png)
+
+Scroll down and choose whatever notification you want to diable.
+
+Shown below is how to disable notifications for warm leads.
+
+For warm leads, there are two types of notification.
+1) Instant notification.
+2) Once-a-day notification.
+
+## Step-6
+
+Disable instant notification.
+
+![image](../../images/Stop-getting-notification-7.png)
+
+If you want to get notified once a day then enable 'once a day notification'.
+
+![image](../../images/Stop-getting-notification-8.png)
+
+## Step-7 
+
+If you want to stop weekly summer notification.
+
+Click on the 'Weekly summer notification' and disable 'notify me'.
+
+![image](../../images/Stop-getting-notification-9.png)
+
+## Step-8
+
+If you want to put processed leads on long-term schedule notification.
+
+(When the schedule is over, lead get into the processed state)
+
+
+Click on 'put processed lead on long term schedule notification' and enable notification.
+
+![image](../../images/Stop-getting-notification-10.png)
+
+## Step-9
+
+How to find the email address of an assistant when a lead is added from EmailCC notification
+
+Go to the assistant tab and click on Copy email.
+
+![image](../../images/Stop-getting-notification-11.png)
+
+Whenever you're using email, you can put assistant email in CC.
+
+You'll get a notification in your inbox when you've put an assistant email in CC.
+
+## Step-10
+
+Link click notification is a feature that notifies you whenever someone is clicking the link you've added in the 'trigger when' section.
+
+To enable this
+
+Click on 'Link click notification', paste your link in the 'Trigger when' section, and enable notification.
```

Comment:
Note the text of this in the UI. It is Link Clicked Notification. Maintain the same.